dietpi/dietpi-software

```diff
@@ -11022,6 +11090,27 @@ _EOF_
 
 		fi
 
+		#HTPC Manager
+		INSTALLING_INDEX=152
+		if (( ${aSOFTWARE_INSTALL_STATE[$INSTALLING_INDEX]} == 1 )); then
+
+			cat << _EOF_ > /etc/systemd/system/htpc-manager.service
+[Unit]
+Description=HTPC Manager
+After=network.target
+
+[Service]
+Type=simple
+ExecStart=/usr/bin/python $FP_DIETPI_USERDATA_DIRECTORY/htpc-manager/Htpc.py
+
+[Install]
+WantedBy=multi-user.target
+_EOF_
+			systemctl daemon-reload
```

Comment:
Not required as both these entries are called at the end of software installations, however, no harm.